internal/plugins/ansible/v1/init.go

```diff
@@ -20,37 +20,47 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/spf13/afero"
 	"github.com/spf13/pflag"
 	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery/util"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 
-	"github.com/operator-framework/operator-sdk/internal/kubebuilder/cmdutil"
 	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1/scaffolds"
-	manifestsv2 "github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2"
-	scorecardv2 "github.com/operator-framework/operator-sdk/internal/plugins/scorecard/v2"
 )
 
+const (
+	groupFlag   = "group"
+	versionFlag = "version"
+	kindFlag    = "kind"
+
+	defaultGroup   = ""
+	defaultVersion = ""
+	defaultKind    = ""
```

Comment:
I do not think that the new consts here are a good idea. 
IMO we need still using "" directly in the bindFlags to avoid misunderstanding.
It makes no sense to create many default const that cannot never assume a value <> of "" empty string. 
Same for helm.

Reply:
https://github.com/operator-framework/operator-sdk/pull/4581#discussion_r588455340